docs/content/topics/operators/security.md

```diff
@@ -19,28 +19,35 @@ For that reason, it is prudent to configure security.
   in a Helm install, do `helm install --namespace brigade ...`.
 - *RBAC Enabled*: When installing with Helm, role-based access control is enabled by default.
   To disable,`--set rbac.enabled=false` will turn off role-based access control.
```

Comment:
This seems a bit off. Even for v1.x, it seems a bit off.

Setting `rbac.enabled=false` won't really _disable_ RBAC. It will prevent roles and rolebindings from being generated and installed, and that. On an RBAC-enabled cluster, setting `rbac.enabled=false` would result in Brigade simply not working.

Probably the best thing to say here is just that if RBAC is enabled in the cluster, then it needs to be enabled here as well. If it's disabled in the cluster, it has to be disabled here as well, although in that case, I think we should _also_ say that we won't vouch for the security of anything Brigade-related in a non-RBAC-enabled cluster.